subprojects/logging/src/test/groovy/org/gradle/internal/logging/console/LogGroupingOutputEventListenerTest.groovy

```diff
@@ -1,137 +0,0 @@
-/*
- * Copyright 2017 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.gradle.internal.logging.console
-
-import org.gradle.api.logging.LogLevel
-import org.gradle.internal.logging.OutputSpecification
-import org.gradle.internal.logging.events.BatchOutputEventListener
-import org.gradle.internal.logging.events.EndOutputEvent
-import org.gradle.internal.logging.events.LogLevelChangeEvent
-import org.gradle.internal.logging.events.OperationIdentifier
-import org.gradle.internal.logging.events.OutputEvent
-import org.gradle.internal.logging.events.ProgressCompleteEvent
-import org.gradle.internal.logging.events.ProgressStartEvent
-import org.gradle.internal.progress.BuildOperationType
-import org.gradle.util.MockExecutor
-import org.gradle.util.MockTimeProvider
-import spock.lang.Subject
-
-class LogGroupingOutputEventListenerTest extends OutputSpecification {
```

Comment:
We'll need a lot more testing coverage to account for all situations. There will also need to be some coverage in the `ConsoleFunctionalTest`.